modules/system/console/OctoberUtil.php

```diff
@@ -328,4 +330,28 @@ protected function utilGitPull()
         }
     }
 
+    /**
+     * Set ProjectId from october.com
+     * Call via: artisan october:util set project --projectId=<id>
+     */
+    protected function utilSetProject(){
+
+        $projectId = $this->option('projectId');
+
+        if(null == $projectId){
```

Comment:
please use `empty($projectId)` instead, and add a space after your `if`

Reply:
resolved in https://github.com/octobercms/october/pull/3946/commits/32d9e14c2e3e2a4c15a3333325a1c6f33bc34c5b